### ISSUE TITLE
Submission TODOs

### ISSUE THREAD

**Issue by devinkwok** [devinkwok/nn_example_difficulty#1]
# Manuscript

## Main text
- [ ] move histograms (fig 2) to appendix
- [ ] shorten intro/related works

## Appendix
- [ ] plot all scores that have been computed already
- [ ] plot same figures for other settings (narrow/wide networks, training hparams)
- [ ] experimental details
- [ ] detailed list of all scores

## Figures
- [x] plot actual score change between different inductive biases: for each score, scatter x-axis model 1 averaged over runs, y-axis model 2 averaged over runs TODO marker size as std
- [x] check distribution of example difficulty scores by class for hard/easy imbalance
- [x] scatter plot distribution of scores of top and bottom most significant examples between different inductive biases: x-axis score 1 (or UMAP-1), y-axis score 2 (or UMAP-2), color: VGG/ResNet/widths/depths etc.
- [x] PCA on variance of scores (replicate existing PCA fig for means, but on variances), and mean/variance combined 
- [ ] select different scores to spotlight, follow dataset cartography paper to plot confidence vs. variability for PCA
- [ ] check plots of all scores, including prediction depth, prototypes, various ensemble scores
- [x] clustering to the top significant examples to separate out different inductive biases without training
- [ ] identify the top significant directions in the clustering

## Literature
- [ ] Representation Matters: Assessing the Importance of Subgroup Allocations in Training Data
- [ ] see if the post-hoc model calibration/outlier detection literature has any relevant scores - these could be included because we have 2 end of training checkpoints

# Code

## Experiments
- [x] run missing replicates CIFAR-10 (lr)
- [x] run missing replicates for CIFAR-100: vgg, resnet, wide resnet
- [x] run replicates for Adam vs SGD
- [ ] use CINIC-10 excluding CIFAR-10 to simulate test split scores, do PCA of mean/variance of those, and combined with train split

## Development
- [ ] plot scores from pretrained ImageNet ResNet and ViT checkpoints

## Optional
- [ ] for all scores averaged over training: compute variance over training time to determine error for a given sampling rate, maybe also split this analysis into some training time bins (e.g. early vs late in training)
- [ ] ResNet on Imagenet capability
- [ ] ViT on Imagenet capability
- [ ] fix implementation of "entropy" score over output logits (not cross entropy loss) - we won't include this score as it requires re-running the experiments
- [ ] try to fix supervised/self-supervised prototypes from pretrained CLIP model: currently the scores are uninformative on CIFAR-10, but it may be fixed by rescaling the images to a larger input size
- [ ] Feldman & Zhang memorization/influence scores

**Comment by devinkwok** [comment]
re post-hoc scores: could compute 1 representative score (do not make it a key contribution)
